### ISSUE TITLE
Release mros_reasoner as a rospy package without ros installation

### ISSUE THREAD

**Issue by chcorbato** [rosin-project/mros1_reasoner#13]
From @gavanderhoorn (thanks!) in #6 

> I would perhaps just try to use rospypi/simple.
> 
> Create a `venv`, just as you've done earlier, run the two install commands shown in the readme -- make sure to install additional packages if/when needed.
> 
> Now install whatever else you need for the reasoner.
> 
> `roslaunch` is probably not needed. The `.launch` file on starts a single node, which you could start as a regular program inside a terminal with the `venv` active.

This is an interesting idea to simplify the release and deployment of mros_reasoner

**Comment by gavanderhoorn** [comment]
I'm not sure I intended for you to *release* `mros_reasoner`. Perhaps my comment wasn't clear.

**Comment by chcorbato** [comment]
No, you are clear, I did not chose the right word, release clearly is not.
build?
develop?

**Comment by chcorbato** [comment]
@marioney are you available to implement this?

**Comment by gavanderhoorn** [comment]
The message dependencies may be a bit tricky to generate btw. If you're current work-around works, you may consider to keep using it.

**Comment by marioney** [comment]
> The message dependencies may be a bit tricky to generate btw.

Do you have / know of an example of this. I've just started looking as you say it seems a bit (lot) tricky.

**Comment by gavanderhoorn** [comment]
See [the readme](https://github.com/rospypi/simple/tree/28ef7383e1d1bf87f0c5f91c02b6678b6046938f#generate-message-python-package).

**Comment by chcorbato** [comment]
Maybe we can postpone this if it is a bit cumbersome, we still have a lot to do for the paper.
